### PR TITLE
use XmlWritter to avoid serialising Xml Declaration in csproj

### PR DESCRIPTION
**Pull request by tebeco**
the idea is to have the minimum amount of change detected in csproj

`<?xml version ...?>`

as never been part of csproj

### CODE REVIEW

File: ElectronNET.CLI/Commands/InitCommand.cs [hunk 1]
```diff
@@ -147,10 +148,19 @@ private static bool EditCsProj(string projectFile)
...
+                };
+                using (XmlWriter xw = XmlWriter.Create(stream, xws))
+                {
+                    xmlDocument.Save(xw);
+                }
 
-                Console.WriteLine($"{ConfigName} added in csproj!");
             }
+
+            Console.WriteLine($"{ConfigName} added in csproj!");
```
Comment (robertmuehsig) [inline ElectronNET.CLI/Commands/InitCommand.cs:163]: Mh - why did you move the Console.WriteLine out of the if(...) section? In theory you can run "init" any time, but we only will add our file once - thats why it is inside the if statement

Comment (tebeco) [reply ElectronNET.CLI/Commands/InitCommand.cs:163]: i wanted to move it out for the top level using block in fact i might have done that too fast (no more computer will update tomorrow)

Comment (tebeco) [reply ElectronNET.CLI/Commands/InitCommand.cs:163]: it was not in any if() section right ? just the last line of the using()
now it is not

Comment (robertmuehsig) [reply ElectronNET.CLI/Commands/InitCommand.cs:163]: AHHH  - whoops - sorry! You are completely right! Looks good!

**Comment by robertmuehsig** [comment]
Will merge it this weekend and release a new NuGet package!

**Comment by robertmuehsig** [comment]
Thanks! Thats great!
